chrome/content/coniks/coniks.js

```diff
@@ -0,0 +1,82 @@
+this.EXPORTED_SYMBOLS = ["coniks"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("resource:///modules/imXPCOMUtils.jsm");
+Cu.import("resource:///modules/jsProtoHelper.jsm");
+Cu.import("chrome://otr/content/coniks/context.js");
+Cu.import("chrome://otr/content/coniks/policies.js");
+Cu.import("chrome://otr/content/helpers.js");
+
+function accountname(acc) {
+  let account = acc.normalizedName;
+  let protocol = acc.protocol.normalizedName;
+  return account+"@"+protocol;
+}
+
+var coniks = {
+  _isEnable: true,
+  get Enabled() { return coniks._isEnable; },
+
+  init: function() {
+    coniks.setConfig();
+    this._contexts = new Map();
```

Comment:
If we're going to keep track of contacts' contexts as well, it might make sense to create a separate context type such as ``accountContext``, which can be seen as an extension of ``coniksContext``, and could take the following form:

``
[
  {
    "account": "alice",
    "protocol/service": Twitter,
    "allowUnsignedKeyChange": false,
    "allowPublicLookup": true,
    "contacts": [coniksContext, coniksContext]
    "tb": null
  }
]
``

What do you think?